test/marked-element.html

```diff
@@ -241,6 +241,21 @@
 
     });
 
+    suite('events', function() {
+      var markedElement;
+
+      setup(function() {
+        markedElement = fixture('CamelCaseHTML');
+      });
+
+      test('render() fires marked-render-complete', function(done) {
+        markedElement.addEventListener('marked-render-complete', function() {
+          done();
```

Comment:
should this have some sort of assert? You're just using the fact that it will timeout?